Create ui-c8y-1021-22-82-response-interval-removed-if-not-needed.md (#3356)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-22-82-response-interval-removed-if-not-needed.md

---
date: ""

> title: Response interval removed when not needed

Reviewer: ```suggestion
title: Response interval value removed when user clears the input
```

> title: Response interval value removed when user clears the input

Reviewer 2: ```suggestion
title: Response interval value deleted when clearing the required interval input
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4743
version: 1021.22.82
---

> Clearing the input now deletes the fragment, so c8y_RequiredAvailability exists only when an interval is actually configured.

Reviewer 2: Same comment from me :-). Please provide some context. Clearing the input where?

> Clearing the input now deletes the fragment, so c8y_RequiredAvailability exists only when an interval is actually configured.

Reviewer: ```suggestion
Clearing the Required Interval input in the Device Info widget now deletes the c8y_RequiredAvailability fragment, so the fragment is present only when an interval is explicitly configured.
```

> Clearing the Required Interval input in the Device Info widget now deletes the c8y_RequiredAvailability fragment, so the fragment is present only when an interval is explicitly configured.

Reviewer 2: ```suggestion
Clearing the **Required interval** input on the **Info** tab in the device details now correctly deletes the c8y_RequiredAvailability fragment, so that the fragment is only present when an interval is explicitly configured.
```

> In some cases, the response interval was shown even when it was not needed, for example, when there was no response expected from the device. This has now been changed so that the response interval is only shown when it is actually needed based on the operation being performed. This change improves clarity for users by not displaying unnecessary information.

Reviewer 2: @dafkk "response interval was shown" Can you please provide some context? Shown where?

> In some cases, the response interval was shown even when it was not needed, for example, when there was no response expected from the device. This has now been changed so that the response interval is only shown when it is actually needed based on the operation being performed. This change improves clarity for users by not displaying unnecessary information.

Reviewer: ```suggestion
Clearing the input now deletes the fragment, so c8y_RequiredAvailability exists only when an interval is actually configured.
```
